src/site/content/en/blog/layoutNG/index.md

```diff
@@ -0,0 +1,234 @@
+
+The example below shows a yellow block sized to content. It uses the Lato font which uses kerning to adjust the spacing between T and -. The bounds of the yellow box should match the bounds of the text.
+
+<div class="w-figures-2-up">
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="kern_legacy.png" alt="trailing whitespace shown at the end of the text container">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 5a, Legacy Layout</small><br>
+      Note the trailing whitespace after the last T
+    </figcaption>
+  </figure>
+
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="kern-ng.png" alt="text bounds have no extra space">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 5b, LayoutNG</small><br>
+      Note how the left and right edges of the box match the bounds of the text
+    </figcaption>
+  </figure>
+</div>
+
+#### Line Wrapping
+Similar to the problem described above, if the content of a size-to-content block is larger (wider) than the block, this can cause the content to wrap unnecessarily. This is quite rare but sometimes happens for mixed directionality content.
+
+<div class="w-figures-2-up">
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="legacy_ar_wrap.png" alt="premature line break shown causing extra space">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 6a, Legacy layout</small><br>
+      Note the unnecessary line break and extra space on the right
+    </figcaption>
+  </figure>
+
+  <figure class="w-figure w-figure--center">
+    <img loading="lazy" src="ng_ar_wrap.png" alt="no unnecessary space or line breaks shown">
+    <figcaption class="w-figcaption w-figcaption--center">
+      <small>Fig 6b, LayoutNG</small><br>
+      Note how the left and right edges of the box match the bounds of the text
+    </figcaption>
+  </figure>
+</div>
+
+<br>
+
+## Further Information
+- [Documentation](https://chromium.googlesource.com/chromium/src/+/master/third_party/blink/renderer/core/layout/ng/README.md)
+- [Layout NG design document](https://docs.google.com/document/d/1uxbDh4uONFQOiGuiumlJBLGgO4KDWB8ZEkp7Rd47fw4/)
+- [Master tracking bug](https://crbug.com/591099)
+
+For more detailed information about the specific compatibility issues and bugs fixed by LayoutNG, please see the issues linked above or search the Chromium bug database for bugs marked [Fixed-In-LayoutNG](https://bugs.chromium.org/p/chromium/issues/list?can=1&q=label%3AFixed-In-LayoutNG).
+
+{% Aside 'warning' %}
```

Comment:
Is a warning the right kind of aside for this? Don't we usually use this to alert readers when some detail will break their code if they don't pay attention to it? Should this be an aside at all?

Reply:
great question, @robdodson have opinions?

my goal was to give visual support for the message (which is a warning), because it's honest, transparent and important to note.